web-client/src/components/NotificationsDrawer/NotificationsDrawer.tsx

```diff
@@ -2,8 +2,8 @@ import { Drawer } from 'antd';
 import { firestore } from 'firebase';
```

Comment:
Changed offers to specificOffers. offers directory is for offers which are responses to requests. In new terminology, these are called specific offers.